instruqt-tracks/azure-cloud-account/01-explore-console/assignment.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: explore-console
+id: tlfoadlgoey6
```

Comment:
For this repository, we should remove id's, this way the track is environment independent. We can then use the CLI to push to either dev or prod with the `--env dev` flag. You can check out the logic in the Workflows.

We can still identify the track with the Track slugs, which should be identical.